src/js_sync.hpp

```diff
@@ -33,6 +33,7 @@
 #include "sync/sync_config.hpp"
 #include "sync/sync_session.hpp"
 #include "sync/sync_user.hpp"
+#include "sync/partial_sync.hpp"
```

Comment:
Is this header needed here? It doesn't look like anything it provides is used here.

Reply:
No, it should be included in `js_realm.hpp` instead.